webapp/src/components/header/settings-dialog/SettingsDialog.tsx

```diff
@@ -93,6 +93,20 @@ export const SettingsDialog: React.FC<ISettingsDialogProps> = ({ open, closeDial
                                     })}
                                 </AccordionPanel>
                             </AccordionItem>
+                            <Divider />
+                            <AccordionItem value="about">
+                                <AccordionHeader expandIconPosition="end">
+                                    <h3>About</h3>
+                                </AccordionHeader>
+                                <AccordionPanel>
+                                    <Body1 color={tokens.colorNeutralForeground3}>
+                                        Frontend version: {process.env.REACT_APP_SK_VERSION ?? "-"}<br/>
+                                        {process.env.REACT_APP_SK_BUILD_INFO}<br/>
+                                        Backend version: {serviceOptions.version}
```

Comment:
nit: can we move this above the commit SHA so that both versions are next to each other?

Reply:
Done